js/cookies.js

```diff
@@ -1,28 +1,63 @@
+// The duration of each cookie
+const duration = 20;
+
+// Save a cookie with indicated duration
 function saveCookie(firstName, lastName, ID)
 {
-    const duration = 20;
 	let date = new Date();
 	date.setTime(date.getTime()+(duration*60*1000));	
 	document.cookie = JSON.stringify({
-        "firstName" : firstName,
-        "lastName" : lastName,
+        "FirstName" : firstName,
+        "LastName" : lastName,
         "ID" : ID,
-        "expires:" : date.toGMTString()
+        "Expires" : date.toGMTString()
     });
 }
 
+// Clears a cookie
 function clearCookie()
 {
     document.cookie = JSON.stringify({
-        "firstName" : -1,
-        "lastName" : -1,
-        "userID" : -1,
-        "expires" : "Thu, 01 Jan 1970 00:00:00 GMT"
+        "FirstName" : -1,
+        "LastName" : -1,
+        "ID" : -1,
+        "Expires" : "Thu, 01 Jan 1970 00:00:00 GMT"
     });
 }
 
+// Returns cookie in JSON format
 function getCookie()
 {
-	let data = document.cookie;
-	return JSON.parse(data);
+	return JSON.parse(document.cookie);
+}
+
+// Returns the validity of a cookie
+function validCookie()
+{
+    let cookie = getCookie();
+    if (cookie["ID"] === -1)
+        return false;
```

Comment:
This is corollary to the comment made above.
```suggestion
    if (cookie === "")
        return false;
```

Reply:
Ah yes this is important, I guess sometimes users don't have a cookie logged.